src/Illuminate/Routing/Router.php

```diff
@@ -238,18 +238,28 @@ public function fallback($action)
     }
 
     /**
-     * Create a redirect from one URI to another.
+     * Create a redirect from one or many URIs to another.
      *
-     * @param  string  $uri
+     * @param  string|string[]  $uri
      * @param  string  $destination
      * @param  int  $status
-     * @return \Illuminate\Routing\Route
+     * @return \Illuminate\Routing\Route|\Illuminate\Routing\Route[]
      */
     public function redirect($uri, $destination, $status = 302)
     {
-        return $this->any($uri, '\Illuminate\Routing\RedirectController')
+        if(is_string($uri)) {
+            $uri = [$uri];
+        }
+
+        $routes = [];
+
+        foreach ($uri as $value) {
```

Comment:
You can do:

```php
foreach ((array) $uri as $value) {
```

To ensure that the string will be cast to an array instead of the longer if check above.

Reply:
Now that's cool. Didn't know. Thanks.